docs/architecture/modernize-desktop/01-why-modern-desktop-applications.md

```diff
@@ -0,0 +1,169 @@
+---
```

Comment:
I'd remove the numbers from the file names

Reply:
Fixed